docbook/Developers_Guide/en-US/Events_Reference_Output.xml

```diff
@@ -150,7 +171,7 @@
 
 				<itemizedlist>
 					<title>Return Value</title>
-					<listitem><para>Array: List of HTML links for the user account menu.</para></listitem>
+					<listitem><para>&lt;Array&gt;: List of HTML links for the user account menu.</para></listitem>
```

Comment:
I'd rather have a look at it separately, to clean some inconsistencies:

For example, EVENT_MENU_ISSUE accepts result in two forms:
   * array of HTML cooked links, eg: "<a href=... >xxxxx</a>"
   * array of key-values as: "link label" => "url target"

And the example of EVENT_MENU_FILTER: only accepts the array of HTML as return.

All menu events could be converted to accept both results, for consistency.
See #0011817 as the example for EVENT_MENU_ISSUE